### ISSUE TITLE
README discussion

### ISSUE THREAD

**Issue by andreimoment** [andreimoment/tangomanual#15]
@mitramartin here are a few things I'd like to include in the README. Let's discuss. 

1. Licensing. Is this going to be under open source license? I feel yes, I recommend the mit license which allows free distribution. https://github.com/angular/angular.js/blob/master/LICENSE
2. What is the recommended use of the manual? Purpose? Efforts behind it? 
3. A mention of P2P learning
4. Possible link to lulu? Explain how the journal is useful?
5. Instructions for contributing (in progress)

**Comment by mitramartin** [comment]
@andreimoment - where does the README appear? will people see it first, before they see the manual? is it at the top of the page? is it possible to include images? where can i edit it?

**Comment by andreimoment** [comment]
Appears as the homepage of the github project. You can edit it by editing
readme.md

**Comment by andreimoment** [comment]
Yes you can add images. See the markdown link in the README.

**Comment by mitramartin** [comment]
@andreimoment will it also appear on the tangomanual website?

**Comment by andreimoment** [comment]
No, this is the readme/introductory file for the tangomanual repository, as seen on the repository home page: https://github.com/andreimoment/tangomanual  (scroll down)

Sounds like you'd like to have a site description on tangomanual.com.

The content of the manual is at tangomanual.com/v1 so technically we could have an introductory page at tangomanual.com.

To create that, edit https://github.com/andreimoment/tangomanual/blob/master/index.md
